packages/cli-tools/bin/streamr-stream-publish.ts

```diff
@@ -63,4 +73,5 @@ createClientCommand(async (client: StreamrClient, streamId: string, options: Opt
     .arguments('<streamId>')
     .description('publish to a stream by reading JSON messages from stdin line-by-line')
     .option('-k, --partition-key-field <string>', 'field name in each message to use for assigning the message to a stream partition')
+    .option('-p, --partition [partition]', 'partition')
```

Comment:
Move the use of `createFnParseInt`here. See `streamr-stream-subscribe.ts` for example.